tests/operators/test_python.py

```diff
@@ -1155,6 +1169,21 @@ def f():
         )
         copy.deepcopy(task)
 
+    def test_except_value_error(self):
+        def f():
+            return 1
+
+        task = PythonVirtualenvOperator(
+            python_callable=f,
+            task_id="task",
+            dag=self.dag,
+        )
+
+        task.log.error = unittest.mock.Mock()
+        task.pickling_library.loads = unittest.mock.Mock(side_effect=ValueError)
+        with pytest.raises(ValueError):
+            task._read_result(path=unittest.mock.Mock())
```

Comment:
Would be a good idea to use a custom exception class to ensure the exception is actually raised by the mocked pickling lib.

Reply:
Created a custom exception class to be raised, which includes the error from `log.error` in its `__str__` method). I might have understood your intentions wrong, let me know if it needs additional fixing.